Core/Src/command_handler.c

```diff
@@ -73,21 +79,34 @@ void take_image() {
  * @param image_count: Pointer to variable containing number of images in NAND fs
  * Untested
  */
-void get_image_count(uint8_t *image_count) {
-    // TODO: Read number of images stored in NAND fs
-    *(image_count) = 0;
-}
+void get_image_count(uint8_t *cnt) { *(cnt) = image_count; }
 
 /**
- * @brief Get the image length
+ * @brief Get the image length fr
  *
  * @param image_length: Pointer to variable containing length of image
```

Comment:
It's not really a pointer to the image length, it's a pointer to a return value. Though I think this function should instead return the length of the image or -1. -1 is not a valid length, so it's clearly an error